test/test_integration_columnar_query.cxx

```diff
@@ -423,7 +451,7 @@ TEST_CASE("integration: columnar query collection does not exist")
   couchbase::core::columnar::agent agent{ integration.io, { { integration.cluster } } };
 
   couchbase::core::columnar::query_options options{ "SELECT * FROM `does-not-exist`" };
-  options.timeout = std::chrono::seconds(20);
+  options.timeout = std::chrono::seconds(60);
```

Comment:
Just curious: does the added seconds have an impact? At first I thought it might be so that it is greater than the dispatch timeout, but that shouldn't have to come into play here I don't think.

Reply:
There wasn't really a reason for this. I've reverted it back to 20 seconds so it's the same with other tests. We might just need to keep an eye on them if it's slower on CI (but 20 seconds should be enough)